paper/build.sh

```
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run --rm  --volume $PWD:/data --user $(id -u):$(id -g) --platform=linux/amd64 --env JOURNAL=joss openjournals/inara
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)